### PR TITLE
update wording on splash page

### PR DESCRIPTION
**Pull request by domdreams**

### CODE REVIEW

File: src/app/components/splash/splash.component.html [hunk 1]
```diff
@@ -1,12 +1,31 @@
 <div class="splashpage">
-  <div class="whorwe">
+  <div class="whorwe" [style.backgroundSize]="'cover'">
       <h1>Who is the Voters' Voice Initiatives?</h1>
-      <p>The Voters’ Voice Initiatives are aimed at providing “we the people” a greater voice in our government. Transparency in campaign finance is critical.  Providing full disclosure on the flow of money in local politics empowers citizens to hold their government accountable for decisions made. Transparency reduces the potential for corruption and increases trust between voters and their government.</p>
+      <p class="vv-description">The Voters’ Voice Initiatives is a non-partisan coalition (including <a href="https://www.lwv.org/">Represent.us</a> and the
```
Comment (nancyheiss) [inline src/app/components/splash/splash.component.html:4]: Looks like these links got swapped.

File: src/app/components/splash/splash.component.html [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
+            <div>
+                <i class="fa fa-handshake-o bullet-icon"></i>
+                <p class="bullet">Reducing the Potential for Corruption</p>
+            </div>
+            <div>
+                <i class="fa fa-gavel bullet-icon"></i>
+                <p class="bullet">Increasing Accountability in Government</p>
+            </div>
+        </div>
       <a routerLink="/under-construction">Learn About Us</a>
```
Comment (nancyheiss) [inline src/app/components/splash/splash.component.html:24]: Do we want to send this one here:  https://sdvotersvoice.org/ ? Or are we building another page for this?

Comment (domdreams) [reply src/app/components/splash/splash.component.html:24]: I was planning on linking it to the about page. Does it make more sense to link it to the site? Should we even have an about page? Maybe we don't need one with this site?

File: src/app/components/splash/splash.component.html (hunk 1 above)
Comment (domdreams) [reply src/app/components/splash/splash.component.html]: Whooops sorry @nancyheiss. Should be fixed now.

File: src/app/components/splash/splash.component.html (hunk 2 above)
Comment (nancyheiss) [reply src/app/components/splash/splash.component.html:24]: I'll ask Amy and see what she says.

Comment (nancyheiss) [reply src/app/components/splash/splash.component.html:24]: @domdreams Amy asked if we could change "Who is the Voters' Voice Initiatives?" to "Who is behind the Voters' Voice Initiatives?"

Comment (domdreams) [reply src/app/components/splash/splash.component.html:24]: @nancyheiss Fixed !

**Pull request merged**